cmdexec/cmdexec.wasm.go

```diff
@@ -0,0 +1,35 @@
+//go:build wasm
+
+// Package `cmdexec` provides a facility to execute
+// l2 commands, stream output to stdout, while also
+// providing ability to retrieve the command output as
+// a string.
+package cmdexec
+
+import (
+	"errors"
+	"fmt"
+
+	"strings"
+
+	"github.com/HexmosTech/httpie-go"
+)
+
+// ExecCommand changes directory to the given `apiDir`
+// and then executes the command specified in `cmdStr`
+// During command execution, ExecCommand streams output
+// to stdout.
+// Once execution finishes, previous CWD is restored,
+// and the command output is returned as a string
+func ExecCommand(cmdSlice []string, stdinBody string) (httpie.ExResponse, error) {
+	proxyURL := "https://proxyserver.hexmos.com/"	
+	proxyUserName := "proxyServer"
+	proxyUserPassword := "proxy22523146server"
```

Comment:
Passwords are open, variables are not declared for the URL, name, password. Is the password meant to be open?